tools/pipelines/templates/include-test-stability.yml

```diff
@@ -0,0 +1,254 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# include-test-stability template to build and test NPM packages/projects
+
+parameters:
+- name: buildDirectory
+  type: string
+
+- name: taskBuild
+  type: string
+  default: ci:build
+
+- name: taskBuildDocs
+  type: boolean
+  default: true
+
+- name: taskLint
+  type: boolean
+  default: true
+
+- name: taskTest
+  type: object
+  default:
+  - ci:test
+
+- name: poolCG
+  type: object
+  default: Small
+
+- name: poolBuild
+  type: object
+  default: Large
+
+- name: checkoutSubmodules
+  type: boolean
+  default: false
+
+- name: buildNumberInPatch
+  type: string
+  default:
+
+- name: nonScopedPackages
+  type: object
+  default: []
+
+- name: publishOverride
+  type: string
+
+- name: releaseBuildOverride
+  type: string
+
+- name: tagName
+  type: string
+
+- name: buildToolsVersionToInstall
+  type: string
+  default: repo
+
+- name: timeoutInMinutes
+  type: number
+  default: 60
+
+jobs:
```

Comment:
This is almost the same as build-npm-package.yml. Can you pull out the common stuff?

Reply:
(Otherwise eliminate the copied stuff that isn't used or never true.)